testing/python/language/test_tilelang_language_sync_threads.py

```
@@ -0,0 +1,45 @@
+    tilelang.disable_cache()
+    try:
+        return tilelang.compile(func, target="cuda", execution_backend="tvm_ffi")
+    finally:
+        tilelang.enable_cache()
+
+
+@tilelang.testing.requires_cuda
+def test_sync_threads_with_variable_barrier_id():
+    @T.prim_func
+    def kernel():
+        with T.Kernel(1, threads=256) as (bx,):
```

Ruff (0.15.12)

[warning] Unpacked variable `bx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)